src/faldbt/project.py

```diff
@@ -71,46 +76,54 @@ def __post_init__(self):
                 self.model = [ref for ref in refs if ref != "where"][0]
                 self.column = node.test_metadata.kwargs.get("column_name", None)
             else:
-                logger.debug(f"Non-generic test was not processed: {node.name}")
+                logger.warn(f"Non-generic test was not processed: {node.name}")
 
 
 @dataclass
 class DbtSource(_DbtNode):
-    name: str = field()
-    table_name: str = field()
-    unique_id: str = field()
-    tests: List[DbtTest] = field(init=False)
+    tests: List[DbtTest] = field(init=False, default_factory=list)
+    status: str = field(init=False, default=None)
 
-    def __post_init__(self):
-        self.tests = []
+    def __repr__(self):
+        attrs = ["name", "tests", "status"]
+        props = ", ".join([f"{item}={repr(getattr(self, item))}" for item in attrs])
+        return f"DbtSource({props})"
```

Comment:
I think this will be auto-generated, even with `init=False` if I am not missing something.

Reply:
It does not work for `@property` functions

Reply:
Makes sense, didn't think about the `property`.